lib/github_core.ml

```diff
@@ -2106,6 +2159,24 @@ module Make(Env : Github_s.Env)(Time : Github_s.Time)(CL : Cohttp_lwt.Client)
       API.get_stream ~rate:Search ?token ~params ~uri (fun b ->
         return [repository_search_of_string b]
       )
+
+   let issues ?token ?sort ?(direction=`Desc) ~qualifiers ~keywords () =
+      let qs = List.rev_map Filter.string_of_issue_qualifier qualifiers in
+      let q = String.concat " " (List.rev_append qs keywords) in
+      let sort = match sort with
+        | Some sort -> Some (Filter.string_of_repo_sort sort)
+        | None -> None
+      in
+      let direction = Filter.string_of_direction direction in
+      let uri = URI.repo_search_issues in
+      let params = [
+        "q", q;
+        "order",direction;
+        "per_page",string_of_int 100;
+      ]@(match sort with None -> [] | Some s -> ["sort",s]) in
+      API.get_stream ~rate:Search ?token ~params ~uri (fun b ->
+        return [repository_issue_search_of_string b]
+      )
```

Comment:
This looks very similar to `Search.repos`. Can the few different functions be abstracted out?